### PR TITLE
[MOB-290] - url schemes for in-app clicked url

### PR DESCRIPTION
**Pull request by tapashmajumder**
1. itbl:// for internal
2. action:// for other

### CODE REVIEW

**Comment by vbabenkoru** [comment]
This should probably target `master`, because it changes the in-app behavior that's already in beta.

**Comment by tapashmajumder** [comment]
Merging to master is going to be a major pain. It is pulling in dozens of commits.

**Comment by vbabenkoru** [comment]
The branch has to be rebased before that

**Comment by tapashmajumder** [comment]
OK. I rebased on master.

File: host-app/ViewController.swift [hunk 1]
```diff
@@ -63,9 +63,13 @@ class ViewController: UIViewController {
...
-        InAppDisplayer.showIterableHtmlMessage(html) { (str) in
-            ITBInfo("callback: \(str ?? "<nil>")")
-            self.statusLbl.text = str
+        InAppDisplayer.showIterableHtmlMessage(html) { (url) in
+            guard let url = url else {
+                ITBError("Could not find url")
+                return
+            }
+            ITBInfo("callback: \(url)")
+            self.statusLbl.text = url.absoluteString
```
Comment (davidtruong) [inline host-app/ViewController.swift]: Minor: I think it's fine to use the full name here for statusLabel

**Pull request merged**